azure-client-authentication/pom.xml

```diff
@@ -6,7 +6,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>com.microsoft.azure</groupId>
+    <groupId>com.microsoft.azure.v2</groupId>
```

Comment:
Let's get Martin and Asir's opinion on this before merging it in. I just want to make sure that they're on board with this.